src/applications/accredited-representative-portal/accreditation/21a/pages/07-character-references-chapter/characterReferencesPages.js

```diff
@@ -36,12 +36,15 @@ const arrayBuilderOptions = {
     !item?.relationship,
   minItems: 3, // TODO: [Fix arrayBuilder minItems validation](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/87155)
   maxItems: 4,
+  showSummaryDescription: true,
```

Comment:
what do you think if we don't introduce a new setting, but the text will show up automatically if it's provided?

Reply:
that seems like a good solution 👍